base/reflectx/values.go

```diff
@@ -951,6 +969,15 @@ func SetRobust(to, from any) error {
 		return fmt.Errorf("destination value cannot be set; it must be a variable or field, not a const or tmp or other value that cannot be set (value: %v of type %T)", pto, pto)
 	}
 
+	// images should not be copied per content: just set the pointer!
+	// otherwise the original images (esp colors!) are altered.
+	if img, ok := to.(*image.Image); ok {
```

Comment:
This actually appears to be a generic issue with setting a pointer to an interface from a non-pointer interface; not sure if this is consistent behavior.

Reply:
Deferring to #1394